.terraform-version

```diff
@@ -0,0 +1 @@
+1.8.5
```

Comment:
This may be a big jump for a lot of existing users. It looks like [the new testing features landed in 1.6](https://developer.hashicorp.com/terraform/language/tests#:~:text=Note%3A%20This%20testing%20framework%20is%20available%20in%20Terraform%20v1.6.0%20and%20later.). (You'd need 1.7 if you were using mocks or passing outputs from one provider into another, but you're not.) So maybe just ask for 1.6.6 here to better reflect the real minimum?

Reply:
I think 1.8.5 is fine here, since that's just the requirement for using terraform from this directory - like when running tests, and not the requirement for use of the modules. That is still specified as `~> 1.0` for now

Reply:
also, now we're using mocks

Reply:
I saw, awesome! Definitely gonna need a download from you when I get back to the module(s) I'm working on.

Reply:
[OpenTofu 1.8.0-alpha1 adds support for mocks!](https://opentofu.org/blog/help-us-test-opentofu-1-8-0-alpha1/#resource-overrides-in-tofu-test)

Reply:
I'm wrong, that's support for resource overrides, which was [a different Terraform 1.7 addition](https://www.hashicorp.com/blog/terraform-1-7-adds-test-mocking-and-config-driven-remove#:~:text=%7D-,Overrides,-In%20addition%20to). Hopefully mocking providers is coming!🤞